platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -21,9 +21,9 @@ namespace mbgl {
 <% if (layoutProperties.length) { -%>
 <% for (const property of layoutProperties) { -%>
```

Comment:
We’ve been relying on an implementation detail of v8 whereby an object’s properties are enumerated in the order that they’re added to the object. Unfortunately, all the renamed properties appear last in the generated source code, which is a little jarring.

To fix this issue, we’d need to sort `layoutProperties`’ keys before enumerating the object:

```js
let sortedLayoutProperties = _.pick(layoutProperties, _.keysIn(layoutProperties).sort());
```

There are probably more efficient ways of doing this, but this way involves less typing. 😉